index.html

```diff
@@ -81,100 +72,125 @@
-          such as in the Unicode Common Locale Data Repository or "CLDR" project
-          [[!CLDR]].</p>
-        <p>Locales can be identified in several ways, which generally are
-          dependent on the programming language and operating environment of the
-          user. One method is by inference from language tags. For example, an
-          implementation could map a language tag from an existing protocol,
-          such as HTTP's Accept-Language header, to its locale model. Locales
-          may also be identified directly by using the language tag syntax in
-          data items (elements, attributes, headers, etc.) that explicitly serve
-          the purpose of locale identification.</p>
-      </section>
-      <section id="out_of_scope">
-        <h3>Out of Scope</h3>
-        <p>This specification does not deal with formats for locale data or
-          actual locale data. One source of locale data and data formats is the
-          Unicode Common Locale Data Repository project ([[CLDR]]).</p>
-      </section>
+
+        <p>Tags for identifying the <a>natural language</a> of content or the <a>international preferences</a> of users (in order to select the natural language of content) are one of the fundamental building blocks of the Web. The <a>language tags</a> found in Web and Internet formats and protocols are defined by [[BCP47]]. Consistent use of language tags provides applications the ability to perform language-specific formatting or processing. For example, a user-agent might use the language to select an appropriate font for displaying text or a Web page designer might style text differently in one language than in another.</p>
+        
+        <p><a>Language tags</a> can also be used to identify <a>international preferences</a> associated with a given piece of content or user because these preferences are linked to the natural language, regional association, or culture of the end user. Such preferences are applied to processes such as presenting numbers, dates, or times; sorting lists linguistically; providing defaults for items such as the presentation of a calendar, or common units of measurement; selecting between 12- vs. 24-hour time presentation; and many other details that users might find too tedious to set individually. Collectively, these preferences are usually called a <a>locale</a>.</p>
+        
+        <p>This document describes best practices for the adoption and use of [[BCP47]] language tags for the identification of <a>natural language</a> content as well as the use of language tags to represent the locale preferences of the user or content author. It describes how document formats, specifications, and implementations should handle the language tags described by [[BCP47]], as well as data structures that extend these tags to describe <a>international preferences</a> (see also sec. 3.1 in [[WS-I18N-SCENARIOS]]). </p>
```

Comment:
Since later in the LTLI draft, there is a definition of international preferences, maybe the reference to WS-I18N-SCENARIOS should be moved to that section, or even dropped?

Reply:
Good catch. I removed the reference, since my intention is for LTLI to be the "standard bearer" for these definitions going forwards.